src/app/project-notification/documents/project-notification-document-table-rows/project-notification-document-table-rows.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input, Output, OnInit, EventEmitter, ChangeDetectorRef } from '@angular/core';
+
+import { TableComponent } from 'app/shared/components/table-template/table.component';
+import { TableObject } from 'app/shared/components/table-template/table-object';
+import { Router, ActivatedRoute } from '@angular/router';
+import { MatSnackBar } from '@angular/material';
+
+@Component({
+  selector: 'tbody[app-document-table-rows]',
+  templateUrl: './project-notification-document-table-rows.component.html',
+  styleUrls: ['./project-notification-document-table-rows.component.scss']
+})
+
+export class PnDocumentTableRowsComponent implements OnInit, TableComponent {
+  @Input() data: TableObject;
+  @Output() selectedCount: EventEmitter<any> = new EventEmitter();
+
+  public documents: any;
+  public paginationData: any;
+  public activeLegislationYear: number;
+
+  constructor(
+    private router: Router,
+    private snackBar: MatSnackBar
+  ) { }
+
+  ngOnInit() {
+    this.documents = this.data.data;
+    this.paginationData = this.data.paginationData;
+    this.activeLegislationYear = 2018;
```

Comment:
Just confirming that this default year is correct?

Reply:
Good question. I'll check

Reply:
It is correct